tests/pytorch_pfn_extras_tests/runtime_tests/test_jit_runtime.py

```diff
@@ -0,0 +1,95 @@
+import types
+import warnings
+
+import torch
+import torch.nn.functional as F
+
+import pytorch_pfn_extras as ppe
+from pytorch_pfn_extras.onnx._as_output import trace
+
+
+class JITRuntime(ppe.runtime.PyTorchRuntime):
+
+    def move_module(self, module):
+
+        def new_forward(self, *args):
+            if hasattr(self, '_traced_mod'):
+                out = self._traced_mod(*args)
+                inter_size = len(self._names)
+                if inter_size == 0:
+                    return out
+                if not isinstance(out, (tuple, list)):
+                    out = [out]
+                return dict(
+                    **{str(i): x for i, x in enumerate(out[:-inter_size])},
+                    **{name: x for name, x in zip(self._names, out[-inter_size:])},
+                )
+
+            new_forward = self.forward
+            self.forward = self._orig_forward
+
+            with trace(self) as (new_module, outputs):
+                warnings.simplefilter('ignore')
+                self._traced_mod = torch.jit.trace_module(new_module, {"forward": args})
```

Comment:
Fixes https://github.com/pfnet/pytorch-pfn-extras/pull/406#discussion_r748066213.
```suggestion
                warnings.simplefilter('ignore')
                self._traced_mod = torch.jit.trace_module(new_module, {"forward": args})
                warnings.simplefilter('error')
```